src/core/Toast/Toast.md

```diff
@@ -0,0 +1,14 @@
+```js
+import { Toast } from 'suomifi-ui-components';
+
+<>
+  <Toast smallScreen>
```

Comment:
Hmm.. there doesn't seem to be `smallScreen` prop for Toast. So this should be removed if there is no need for the actual different version for small screen, that is.

Reply:
Removed smallScreen